modules/javafx.base/src/test/java/test/javafx/collections/ObservableListTest.java

```diff
@@ -255,6 +255,26 @@ public void testSet() {
         assertEquals("two", r);
     }
 
+    @Test
+    public void testSetAll() {
+        useListData("one", "two", "three");
+        boolean r = list.setAll("one");
+        assertTrue(r);
+
+        r = list.setAll("one", "four", "five");
+        assertTrue(r);
+
+        r = list.setAll();
+        assertTrue(r);
```

Comment:
Can you also test that calling `setAll` when the list is currently empty returns true? Repeating one of the earlier checks should work:

```
        r = list.setAll("one");
        assertTrue(r);
```

Reply:
Sure. Added in #284 5de1a51fd3a8d744acf0b831ab4f5a0441e417d7.